pkg/provisioner/provision.go

```diff
@@ -57,7 +82,13 @@ func (p ZFSProvisioner) createVolume(options controller.VolumeOptions) (string,
 	zfsPath := p.parent.Name + "/" + options.PVName
 	properties := make(map[string]string)
 
-	properties["sharenfs"] = p.shareOptions
+	if p.exportNfs {
```

Comment:
This if block should only do things with  the logs, p.shareOptions has already been correctly set to the nfs options or "off" before now.

Reply:
Fixed!